config/credentials/production.yml.enc

```diff
@@ -0,0 +1 @@
+N3M0tKvmR0uUcdEp/oNA02rHCOXwZY2CC/lvtbie1btkT8bjzx6o3gYTce/XjUqZbqTydgi/TtwZFspNaMA2wMTlxBwEGLwFY+WqFwBKHFK1cw0HqRqAVspVOTRHHYURnzzxt3IqpaQBm/pR4g6ypqa4q0vQlYFBow6EZOvI0XQckdcoqIZvrd3l5GQuKXBpTAHfIcES/NDCUN8rPuhEyOtzHvgIMTRkoEPK21nVtM8MC8ZM8aPh/xpysIUdyJ9M09Hyd2Q+B96RMdmPV5IQvZUwePjun9IxOb4lyIen6iI49hXjimo6W0ct6cXKhLFF9n9YyInYwZcZlLkE3OThiZsgqUluM/hxz93Cn9De9pF85/Ju/UpfqNdYuRbOo1IRMizSYbXihb3UhUwaSIo2TWhoH9y6zCTFqU7fPOK+K4tr3hPGyQxSQaqol0+yiFOlWNGs3UtFhZtUILkqV1WBaDelwzW92gQGEqK7RteXaFVrAVFbbftwdgT1jJgSvLTHml6ysnOZNRwjeskiZWxHa2DhiKQQb81b//r5ma7LPU7Nfli0IyMokbWumzN1cTq44bjTk5qSa0lzREtLGkKFLVJiP+HJ/EaHu+H+DRBlRVQYO1G4vZ4ECBJzcfFxpU1ClyPbwf4IurW/--ZmNjs3R4MOx9x1hJ--ArdhnAr6jbIXpqgKEIcsyQ==
```

Comment:
New goodness for the credentials for deployment. Please update this when we go real production.